app/src/screens/ActiveCall/SpeakerButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Image, StyleSheet } from 'react-native';
+import Button, { type Props as ButtonProps } from '../../components/Button';
+
+const SpeakerSource = require('../../../assets/icons/speaker.png');
+const ActiveSpeakerSource = require('../../../assets/icons/speaker-active.png');
+
+export type Props = {
```

Comment:
Do we import this from other files? I cannot find where this is being imported, unless I'm missing it. Also, I realized we follow this same pattern on most files..

Reply:
We don't import these in other files yet. In the past with other components I've found it necessary sometimes to be able to get the Props of another component (especially when extending the component) so I do it by default. If we want to remove the exports we can, but they don't hurt.

I also find that exporting as `Props` makes more since than `SpeakerButtonProps` because you import like so
```
import { Props } from './SpeakerButton';
```
It's very clear it's props for the speaker button, and having it named `SpeakerButtonProps` is redundant. If there's a naming clash (like you need to import props from multiple components) you can just do this
```
import { Props as SpeakerButtonProps } from './SpeakerButton';
```

Reply:
I know they don't hurt, but usually when I see exports I expect them to be used somewhere else.

Reply:
I think we should not export things that are not used outside of the file. We can export it once we need it. We also missed this last time. It seems we are already exporting a bunch of these things that don't need to be exported. I'm ok to keep it as is but we need to have a todo or ticket to stop exporting these.

Reply:
Ok, but curious as to why it's an issue. We don't export it out of the project for the consuming application, so it's not visible to the end developer. And it's a type, so it's not mutable or anything dangerous.

Reply:
But I do know that it's best practice typically to not needlessly export things, so I will not continue this pattern going forward.

Reply:
One, just good practice. If you don't need to export, don't export, preventing future mistakes in case we need to encapsulate something. The other thing is, just like what kevin mentioned, it adds confusion. If you see an export keyword here, I would assume it's being used somewhere so changing it might be more risky since I wouldn't want other files to break.